docs/app-store-connect/get-latest-build-number.md

```diff
@@ -0,0 +1,102 @@
+
+get-latest-build-number
+=======================
+
+
+**Find the highest build number used for given app considering both TestFlight and App Store submissions**
```

Comment:
same as above, can potentially ignore `find -> get`, but extra `the` would be nice

```suggestion
**Get the highest build number used for the given app considering both TestFlight and App Store submissions**
```